mappings/net/minecraft/client/gui/widget/ScrollableWidget.mapping

```diff
@@ -0,0 +1,45 @@
+CLASS net/minecraft/unmapped/C_djvyunwn net/minecraft/client/gui/widget/ScrollableWidget
+	COMMENT A widget that can be focused and vertically scrolled.
+	FIELD f_awqtdtsd UNFOCUSED_BORDER_COLOR I
+	FIELD f_dbzjuacb scrollbarDragged Z
+	FIELD f_djpqddjv BOX_BACKGROUND_COLOR I
+	FIELD f_kciammnc scrollAmount D
+	FIELD f_mtmzekuk FOCUSED_BORDER_COLOR I
+	FIELD f_uajaeyqh INNER_PADDING I
+	METHOD m_arnarrjg getTotalInnerPadding ()I
+	METHOD m_eucbjkoi renderContents (Lnet/minecraft/unmapped/C_cnszsxvd;IIF)V
+		COMMENT Renders the scrolled contents.
+		COMMENT Subclasses must override this.
+		COMMENT <p>
+		COMMENT The rendered contents may overflow and will be automatically be trimmed.
+		ARG 2 mouseX
+		ARG 3 mouseY
+		ARG 4 delta
+	METHOD m_eyjlocli isWithinBounds (DD)Z
+		ARG 1 x
+		ARG 3 y
+	METHOD m_fbsebolr drawBox (Lnet/minecraft/unmapped/C_cnszsxvd;)V
+		COMMENT Draws the box that the contents are rendered over, including its borders.
+			COMMENT the matrix stack
```

Comment:
Something looks wrong here :thinking:

Reply:
I... uh... what...?

Enigma you good?
Why did it just decided to put a COMMENT there??
Where's the ARG???
Is it because it's auto-mapped to matrices and I haven't made it explicit??

Reply:
Yeah that's a problem... Should be fairly easy to fix though